src/axom/primal/operators/compute_bounding_box.hpp

```diff
@@ -169,8 +170,8 @@ template <typename T, int NDIMS>
 AXOM_HOST_DEVICE BoundingBox<T, NDIMS> compute_bounding_box(
   const Polyhedron<T, NDIMS> &poly)
 {
-  BoundingBox<T, NDIMS> res(poly[0]);
-  for(int i = 1; i < poly.numVertices(); i++)
+  BoundingBox<T, NDIMS> res;
+  for(int i = 0; i < poly.numVertices(); i++)
```

Comment:
😮

Reply:
Looks like a premature optimization that assumed a polyhedron would have at least one vertex.
I think it was copy/pasted from previous code that did the same for Triangles/Tetrahedra (before we used initializer lists), which always have 3 or 4 vertices, respectively.

Reply:
Nevertheless, it's a very good catch!